src/Core/src/Handlers/Entry/EntryHandler.iOS.cs

```diff
@@ -96,14 +101,14 @@ public static void MapClearButtonVisibility(IEntryHandler handler, IEntry entry)
 
 		public static void MapFormatting(IEntryHandler handler, IEntry entry)
 		{
-			handler.PlatformView?.UpdateMaxLength(entry);
```

Comment:
This wasn't taking into consideration that a developer can provide their own mappers.